src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -0,0 +1,290 @@
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to read [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% /section %}
+{% section #step-2 step=2 title="Create Go project" %}
+
+Create a go application.
+
+```sh
+mkdir my-app
+cd my-app
+go mod init go-appwrite/main
+```
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Go Appwrite SDK.
+
+```sh
+go get github.com/appwrite/sdk-for-go
+```
+
+{% /section %}
+
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier.
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Create a new file called `app.go`, initialize a function, and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```go
+package main
+
+import (
+	"github.com/appwrite/sdk-for-go/appwrite"
+)
+
+func main() {
+	client := appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_ID>"),
+		appwrite.WithKey("<API_KEY>"),
+	)
+}
+```
+
+{% /section %}
+{% section #step-5 step=5 title="Initialize database" %}
+
+Once the Appwrite Client is initialized, create a function to configure a todo collection.
+
+```go
+func todoCollection(client client.Client) {
+	appwriteDatabases := appwrite.NewDatabases(client)
+
+	todoDatabase, err := appwriteDatabases.Create("unique()", "TodosDB")
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	todoCollection, err := appwriteDatabases.CreateCollection(
+		todoDatabase.Id,
+		id.Unique(),
+		"Todos")
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	attribute1, err := appwriteDatabases.CreateStringAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"title",
+		255,
+		true,
+	)
+
+	fmt.Println(attribute1)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	attribute2, err := appwriteDatabases.CreateStringAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"description",
+		255,
+		false,
+	)
+
+	fmt.Println(attribute2)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	attribute3, err := appwriteDatabases.CreateBooleanAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"isComplete",
+		true,
+	)
+
+	fmt.Println(attribute3)
```

Comment:
Let's not print these because they're not really comprehensible:
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/932a9306-0d72-411b-98f9-33c28be4b29c">